tests/unit/modules/test_rpm_lowpkg.py

```diff
@@ -1,478 +0,0 @@
-GwMGCwkIBwMCBBUCCAMEFgIDAQIeAQIXgAUCU2dN1AUJHR8ElQAKCRC4iy/UPb3C
-hGQrB/9teCZ3Nt8vHE0SC5NmYMAE1Spcjkzx6M4r4C70AVTMEQh/8BvgmwkKP/qI
-CWo2vC1hMXRgLg/TnTtFDq7kW+mHsCXmf5OLh2qOWCKi55Vitlf6bmH7n+h34Sha
-Ei8gAObSpZSF8BzPGl6v0QmEaGKM3O1oUbbB3Z8i6w21CTg7dbU5vGR8Yhi9rNtr
-hqrPS+q2yftjNbsODagaOUb85ESfQGx/LqoMePD+7MqGpAXjKMZqsEDP0TbxTwSk
-4UKnF4zFCYHPLK3y/hSH5SEJwwPY11l6JGdC1Ue8Zzaj7f//axUs/hTC0UZaEE+a
-5v4gbqOcigKaFs9Lc3Bj8b/lE10Y
-=i2TA
------END PGP PUBLIC KEY BLOCK-----
-
-Distribution: (none)
-"""
-        mock = MagicMock(return_value=info)
-        with patch.dict(rpm.__salt__, {"cmd.run_stdout": mock}):
-            self.assertEqual(
-                rpm.info_gpg_key("key"),
-                {
-                    "Name": "gpg-pubkey",
-                    "Version": "3dbdc284",
-                    "Release": "53674dd4",
-                    "Architecture": None,
-                    "Install Date": datetime.datetime(2019, 3, 8, 11, 57, 44),
-                    "Group": "Public Keys",
-                    "Size": 0,
-                    "License": "pubkey",
-                    "Signature": None,
-                    "Source RPM": None,
-                    "Build Date": datetime.datetime(2014, 5, 5, 10, 37, 40),
-                    "Build Host": "localhost",
-                    "Packager": "openSUSE Project Signing Key <opensuse@opensuse.org>",
-                    "Summary": "gpg(openSUSE Project Signing Key <opensuse@opensuse.org>)",
-                    "Description": """-----BEGIN PGP PUBLIC KEY BLOCK-----
-Version: rpm-4.14.2.1 (NSS-3)
-
-mQENBEkUTD8BCADWLy5d5IpJedHQQSXkC1VK/oAZlJEeBVpSZjMCn8LiHaI9Wq3G
-3Vp6wvsP1b3kssJGzVFNctdXt5tjvOLxvrEfRJuGfqHTKILByqLzkeyWawbFNfSQ
-93/8OunfSTXC1Sx3hgsNXQuOrNVKrDAQUqT620/jj94xNIg09bLSxsjN6EeTvyiO
-mtE9H1J03o9tY6meNL/gcQhxBvwuo205np0JojYBP0pOfN8l9hnIOLkA0yu4ZXig
-oKOVmf4iTjX4NImIWldT+UaWTO18NWcCrujtgHueytwYLBNV5N0oJIP2VYuLZfSD
-VYuPllv7c6O2UEOXJsdbQaVuzU1HLocDyipnABEBAAG0NG9wZW5TVVNFIFByb2pl
-Y3QgU2lnbmluZyBLZXkgPG9wZW5zdXNlQG9wZW5zdXNlLm9yZz6JATwEEwECACYC
-GwMGCwkIBwMCBBUCCAMEFgIDAQIeAQIXgAUCU2dN1AUJHR8ElQAKCRC4iy/UPb3C
-hGQrB/9teCZ3Nt8vHE0SC5NmYMAE1Spcjkzx6M4r4C70AVTMEQh/8BvgmwkKP/qI
-CWo2vC1hMXRgLg/TnTtFDq7kW+mHsCXmf5OLh2qOWCKi55Vitlf6bmH7n+h34Sha
-Ei8gAObSpZSF8BzPGl6v0QmEaGKM3O1oUbbB3Z8i6w21CTg7dbU5vGR8Yhi9rNtr
-hqrPS+q2yftjNbsODagaOUb85ESfQGx/LqoMePD+7MqGpAXjKMZqsEDP0TbxTwSk
-4UKnF4zFCYHPLK3y/hSH5SEJwwPY11l6JGdC1Ue8Zzaj7f//axUs/hTC0UZaEE+a
-5v4gbqOcigKaFs9Lc3Bj8b/lE10Y
-=i2TA
------END PGP PUBLIC KEY BLOCK-----""",
-                    "Distribution": None,
-                },
-            )
-            self.assertFalse(_called_with_root(mock))
-
-    def test_remove_gpg_key(self):
-        """
-        Test remove_gpg_key
-        """
-        mock = MagicMock(return_value=0)
-        with patch.dict(rpm.__salt__, {"cmd.retcode": mock}):
-            self.assertTrue(rpm.remove_gpg_key("gpg-pubkey-1"))
-            self.assertFalse(_called_with_root(mock))
```

Comment:
I think we should move these tests to the new pytests file. These are not included in the pytest file from the upstream PR, as these tests are probably coming from one of our downstream patches for 3002.2.

Reply:
You mean the tests related to gpg, right? I'll move them.

Reply:
Yeah, exactly. Thanks! :+1:

Reply:
I just pushed https://github.com/openSUSE/salt/pull/448/commits/d9a0120c61f49b4f8f09c21facb7859e8f237dc5 moving the GPG tests back and realigns them to pytest